README.md

```diff
@@ -296,6 +296,18 @@ parameter "[HOST]", "server address", :environment_variable => "MYAPP_HOST"
 
 Clamp will check the specified envariables in the absence of values supplied on the command line, before looking for a default value.
 
+### Allowing options after parameters
+
+Many option-parsing libraries - notably [GNU `getopt(3)`](https://www.gnu.org/software/libc/manual/html_node/Using-Getopt.html) - allow option and parameter arguments to appear in any order on the command-line, e.g.
+
+    foobar --foo=bar something --fnord=snuffle another-thing
+
+By default, Clamp does not allow options and parameters to be "interspersed" in this way. If you want that behaviour, set:
```

Comment:
I think ```that``` is a bit vague in this context. Alternative:
> ... in this way but has an option to allow it:

Reply:
Fair point, @ekohl; I'll edit to clarify.

Reply:
That's easier to parse :)